sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/SqlQueryContext.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.trees
+
+import org.apache.spark.QueryContextSummary
+
+/** The class represents error context of a SQL query. */
+case class SqlQueryContext(
```

Comment:
nit: we usually upper case `SQL`, e.g. `SQLConf`, `SQLContext`, `SQLExecution`

Reply:
ok. I will rename it. Just in case, I have found the classes in code base that start from `Sql`:
- SqlResource
- SqlBasedBenchmark
- SqlStatsTableDataSource
- SqlStatsTableRow
- SqlStatsPagedTable
- SqlCmd
- SqlResourceSuite
- SqlResourceWithActualMetricsSuite